### ISSUE TITLE
Window closing button color contrast issue

### ISSUE THREAD

**Issue by MaximPerry** [MaximPerry/Desktop-Timer#6]
The window's "x" closing button fails WCAG colour contrast criteria.

**Comment by MaximPerry** [comment]
Adapted colours to be WCAG compliant. Closing issue.
